app/models/setting/katello.rb

```diff
@@ -20,7 +20,9 @@ def self.load_defaults
         self.set('check_services_before_actions', N_("Whether or not to check the status of backend services such as pulp and candlepin prior to performing some actions."), true),
         self.set('force_post_sync_actions', N_("Force post sync actions such as indexing and email even if no content was available."), false),
         self.set('default_download_policy', N_("Default download policy for repositories (either 'immediate', 'on_demand', or 'background')"), "immediate"),
-        self.set('pulp_export_destination', N_("On-disk location for exported repositories"), "/tmp/katello-repo-exports")
+        self.set('pulp_export_destination', N_("On-disk location for exported repositories"), "/tmp/katello-repo-exports"),
+        self.set('pulp_client_key', N_("Path for ssl key used for pulp server auth"), "/etc/pki/katello/key.key"),
+        self.set('pulp_client_cert', N_("Path for ssl cert used for pulp server auth"), "/etc/pki/katello/server.crt")
```

Comment:
I totally forgot about this :flushed: If there is no reason we need a hostname in the filename,  I think it would be easier and lead to less problems to modify the installer to create a fixed name for the cert and key. I can make this change